donut Util changes (#216)
patternfly/training-scenarios: react-charts/donut-utilization-chart/step2.md

Create a simple donut utilization chart.

1) <strong>Navigate to the `src` folder and open `src/App.js`{{open}}</strong>

2) <strong>Set up the general structure of the chart and the necessary import statements.</strong>

> Copy the following code into the App.js file, replacing all of the content there:

Reviewer: back-ticks

<pre class="file" data-filename="App.js" data-target="replace">
import React from 'react';
import "@patternfly/react-core/dist/styles/base.css";
import './app.css';
import {
  ChartAxis,
  ChartDonutThreshold,
  ChartDonutUtilization,
  ChartThemeColor
} from '@patternfly/react-charts';

function App() {
  return (
    &lt;div className=&quot;chart-container&quot;&gt;
      &lt;ChartDonutUtilization
        data={{ x: &#39;Storage capacity&#39;, y: 90 }}
        title=&quot;90%&quot;
        subTitle=&quot;of 100 GBps&quot;
        height={230}
        width={500}
      /&gt;
    &lt;/div&gt;
  );
}

export default App;
</pre>

> When the server reloads, you should see something like this:

Reviewer: ```suggestion
Once the preview reloads, it should look like this:
```
